framework/tests/all-systems/TypeVars.java

```diff
@@ -3,7 +3,7 @@
 
     class Test1<T> {
         void m() {
-            @SuppressWarnings("unchecked")
+            @SuppressWarnings({"unchecked", "assignment.type.incompatible"})
```

Comment:
This assignment works if the default for local variables is `@Top`.  The Unit Checker issued an assignment.type.incompatible error.  (The Nullness Checker does not issue a warning here and the cast type is `@NonNull T`.)